### PR TITLE
BOOKING-132: Added user-bookings endpoint for retrieving user booking…

### PR DESCRIPTION
**Pull request by jeppekroghitk**
…s from userId

### CODE REVIEW

**Comment by jeppekroghitk** [comment]
TODO: Opret tests

File: src/Service/MicrosoftGraphService.php [hunk 1]
```diff
@@ -291,4 +291,20 @@ public function getUserBookings(string $userId): array
...
+    /**
+     * @throws GuzzleException|GraphException
+     *
+     * @see https://docs.microsoft.com/en-us/graph/search-concept-events
+     */
+    public function getBookingDetails(string $bookingId): array
+    {
+        $token = $this->authenticateAsServiceAccount();
+
+        $bookingId_formatted = str_replace(['/', ' '], ['-', '+'], $bookingId);
```
Comment (tuj) [inline src/Service/MicrosoftGraphService.php:304]: why is this needed. Please add a comment.

**Review by tuj — APPROVED** [review]

**Comment by jeppekroghitk** [comment]
@tuj kan du reviewe de nye changes potentielt merge igen i det samme PR, selvom det er merget tidligere?
